nautobot/dcim/templates/dcim/device/base.html

```diff
@@ -8,6 +8,28 @@
 {% block title %}{{ object }}{% endblock %}
 
 {% block header %}
+<style>
+    span.ip_hover_button {
+        display: none;
+    }
+
+    span.ip_hover:hover span.ip_hover_button {
+        display: inline;
+    }
+</style>
+<script src="{% static 'clipboard.js/clipboard-2.0.6.min.js' %}"
+    onerror="window.location='{% url 'media_failure' %}?filename=clipboard.js/clipboard-2.0.6.min.js'"></script>
+    <script>
+        var clipboard = new ClipboardJS('.btn');
+
+        clipboard.on('success', function (e) {
+            console.log(e);
+        });
+
+        clipboard.on('error', function (e) {
+            console.log(e);
+        });
+    </script>
```

Comment:
This should ideally be in `{% block javascript %}` (default block provided by the `base.html` template; add it to the end of this file if not already present), not `{% block header %}`.